### PR TITLE
integrate altair IDE

### PR DESCRIPTION
**Pull request by ankitjena**
### Description

Integrate altair IDE as playground for the ionic-showcase server

### CODE REVIEW

File: server/src/index.js [hunk 1]
```diff
@@ -46,6 +47,10 @@ async function start() {
   applyFileMiddelware(app);
   app.get('/health', (req, res) => res.sendStatus(200))
 
+  app.use('/graphql', altairExpress({
+    endpointURL: '/graphql'
```
Comment (wtrocki) [inline server/src/index.js]: Is there a way to pass queries that we have defined in `config.playgroundConfig`? 

See `apolloConfig.playground` (we should probably disable that as well)

File: server/src/config/config.js [hunk 2]
```diff
@@ -33,6 +33,11 @@ class Config {
         }
       ]
     }
+
+    this.altairConfig = {
+      endpointURL: '/graphql',
+      initialQuery: fs.readFileSync(path.resolve(__dirname, './playground.gql'), 'utf8').replace(/\n/g, ' '),
```
Comment (wtrocki) [inline server/src/config/config.js]: Replace could fail if file is missing. 
We should also disable playground config itself as having two playgrounds in the single app makes no sense.

File: server/src/index.js [hunk 3]
```diff
@@ -47,17 +47,14 @@ async function start() {
...
-  }))
+  app.use('/graphql', altairExpress(config.altairConfig))
 
   // connect to db
   const client = await connect(config.db);
 
   const apolloConfig = {
     typeDefs: appTypeDefs,
     resolvers: appResolvers,
-    playground: config.playgroundConfig,
```
Comment (wtrocki) [inline server/src/index.js]: If we run project without altairExpress above playground still appear. 
We missing required flags: https://www.apollographql.com/docs/apollo-server/features/graphql-playground.html#Enabling-GraphQL-Playground-in-production

File: server/src/config/config.js (hunk 2 above)
Comment (ankitjena) [reply server/src/config/config.js]: We were importing from a file right? Taking all the initial queries from a file seems like a good practice.

Comment (wtrocki) [reply server/src/config/config.js]: I meant that this file may not exist or be removed. However let's ignore that for the moment.

Comment (ankitjena) [reply server/src/config/config.js]: I have made the other changes.

**Comment by wtrocki** [comment]
@ankitjena This looks very good now! I will need some time to play with this and let you know if I see some issues (I use that playground on a daily basis).

**Comment by ankitjena** [comment]
Sure, lemme know what else I can work in the meanwhile.

**Comment by wtrocki** [comment]
👀 Checking that now

**Comment by wtrocki** [comment]
@ankitjena It looks like Altair is not fully usable.
When trying to execute anything it will ask for websocketUrl. This URL is known and we can pass that. Additionally, there is no way to execute each individual query.
The only way to execute queries now is to remove other queries from the window. 
Additionally creating a new tab will copy entire content from the previous one. This could be a limitation of the Altair, but maybe we can figure out how to make things work for us.

**Comment by wtrocki** [comment]
I could not see any way to integrate this without limiting what needs to be done. 
While playground is quite good and does not have any bugs it does not allow to run separate queries written in the window. I'm happy to review that again once we figure out how to make it usable.

**Comment by imolorhe** [comment]
@wtrocki If you have multiple queries, Altair would give you the option to choose a query. Alternatively, the query your mouse cursor is in at the time of sending the request would be the query that is sent.

**Comment by wtrocki** [comment]
Interesting. I was expecting some button to drop list of active queries that needs to be executed like here: 

image

Somehow this did not happen. @ankitjena Can you play with this again and see if there will be an option. Happy to integrate that once we get that sorted.

**Pull request merged**